File: Python/Fibonacci.py
def fibonacci(number):
    if number == 0 or number == 1:
        return number
    else:
        return (fibonacci(number - 1) + fibonacci(number - 2))
    
print(fibonacci(1))
print(fibonacci(2))
print(fibonacci(3))
print(fibonacci(4))
print(fibonacci(5))
print(fibonacci(6))

File: JavaScript/Fibonacci.js
const fibonacci = (number) => {
    if (number === 0 || number === 1) {
        return number;
    } else {
        return (fibonacci(number - 1) + fibonacci(number - 2));
    }

}
console.log(fibonacci(1));
console.log(fibonacci(2));
console.log(fibonacci(3));
console.log(fibonacci(4));
console.log(fibonacci(5));
console.log(fibonacci(6));
